apiserver/facades/client/usermanager/usermanager.go

```diff
@@ -269,13 +270,20 @@ func (api *UserManagerAPI) UserInfo(ctx context.Context, request params.UserInfo
 	}
 
 	var accessForUser = func(userTag names.UserTag, result *params.UserInfoResult) {
-		// Lookup the access the specified user has to the controller.
-		access, err := common.GetPermission(api.state.UserPermission, userTag, api.state.ControllerTag())
-		if err == nil {
-			result.Result.Access = string(access)
-		} else if err != nil && !errors.Is(err, errors.NotFound) {
+		userPermission := func(subject names.UserTag, target names.Tag) (permission.Access, error) {
+			access, err := api.accessService.ReadUserAccessForTarget(ctx, subject.Id(), permission.ID{
+				ObjectType: permission.Controller,
+				Key:        "controller",
+			})
+			return access.Access, errors.Trace(err)
+		}
+
+		access, err := common.GetPermission(userPermission, userTag, api.state.ControllerTag())
```

Comment:
There will be a Jira task to remove `GetPermission`, since the access service will check `everyone@external` permissions. That same task will include the change to common `HasPermission`.